src/client/java/teammates/client/scripts/PerformanceProfiler.java

```diff
@@ -497,22 +497,21 @@ public String createStudent() {
         return status;
     }
 
-    /**
-     * The method createSubmission is not implemented in BackDoor yet.
-     * @return
-     */
-//    @PerformanceTest(name = "BD create submission")
-//    static public String createSubmissions()
-//    {
-//        String status = "";
-//        Set<String> set = data.submissions.keySet();
-//        for (String submissionKey : set)
-//        {
-//            SubmissionData submission = data.submissions.get(submissionKey);
-//            status += " " + BackDoor.createSubmission(submission);
-//        }
-//        return status;
-//    }
+    // The method createSubmission is not implemented in BackDoor yet.
+    @PerformanceTest(name = "BD create submission")
+    static public String createSubmissions()
+    {
```

Comment:
The position of the opening bracket should start one space after the params of the method instead of the new line to be consistent with the code base.